source/math/FloatVectorList.ooc

```diff
@@ -46,6 +46,14 @@ FloatVectorList: class extends VectorList<Float> {
 			result
 		}
 	}
+	maxValue: Float {
+		get {
+			result: Float
+			for (i in 0..this count)
+				result = Float maximum(result, this[i])
```

Comment:
This is unnecessary work,
`if (result < this[i]) result = this[i]`
should be more effective.